bot/exts/fun/movie.py

```diff
@@ -107,7 +121,8 @@ async def genres(self, ctx: Context) -> None:
         """Show all currently available genres for .movies command."""
         await ctx.send(f"Current available genres: {', '.join('`' + genre.name + '`' for genre in MovieGenres)}")
 
-    async def get_movies_data(self, client: ClientSession, genre_id: str, page: int) -> list[dict[str, Any]]:
+    async def get_movies_data(self, client: ClientSession, genre_id: str, page: int)\
+            -> tuple[list[dict[str, Any]], int]:
```

Comment:
```suggestion
    async def get_movies_data(
        self,
        client: ClientSession,
        genre_id: str,
        page: int
    ) -> tuple[list[dict[str, Any]], int]:
```
To follow the [style guide](https://www.pythondiscord.com/pages/guides/pydis-guides/contributing/style-guide/)